### ISSUE TITLE
Sniff for detecting Useless Trailing Comma﻿

### ISSUE THREAD

**Issue by o5** [slevomat/coding-standard#1439]
Is there any sniff for detecting this?

```php
class Foo
{
    public function __construct($bar,)
}
```

It should have similar funkcionality like [PHPStorm Inspections](https://www.jetbrains.com/help/phpstorm/php-useless-trailing-comma.html)

**Comment by michnovka** [comment]
Do you mean https://github.com/slevomat/coding-standard#slevomatcodingstandardfunctionsrequiretrailingcommaincall- ?

**Comment by michnovka** [comment]
Or https://github.com/slevomat/coding-standard#slevomatcodingstandardfunctionsdisallowtrailingcommaindeclaration-

**Comment by o5** [comment]
@michnovka Thanks, I known these sniffs. My question was not about require/disallow trailing comma but about detecting where trailing commas are useless. Look at my example, it should be marked as useless trailing comma﻿.

**Comment by michnovka** [comment]
@o5 but check the `SlevomatCodingStandard.Functions.DisallowTrailingCommaInDeclaration` it looks like what you need, itll mark the comma as an error.

**Comment by o5** [comment]
@michnovka I missed that feature added in [this](https://github.com/slevomat/coding-standard/commit/1b70ab56feab02cba4315da4bd5de0569c11198f) commit :-) Thank you!
